fitnessllm_dataplatform/task_handler.py

```
@@ -1,160 +1,201 @@
+
+        if data_source == FitnessLLMDataSource.STRAVA.value:
+            strava_user_data = (
+                self.firebase.read_user()
+                .collection("stream")
+                .document(data_source.lower())
+                .get()
+                .to_dict()
+            )
+
+            if strava_user_data is None:
+                raise ValueError(f"User {uid} has no {data_source} data")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)